ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml

```diff
@@ -1713,3 +1713,16 @@ jobs:
         template: *ci-ipa-4-9-previous
         timeout: 3600
         topology: *master_1repl
+
+  fedora-previous-ipa-4-9/test_custom_plugins:
+    requires: [fedora-previous-ipa-4-9/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
+        selinux_enforcing: True
```

Comment:
The other tests for 4-9-previous don't run in selinux enforcing mode, this line should be removed.

Reply:
Fixed, too quick copy-pasta.